### DISCUSSION TITLE
Test using given executable/command?

### DISCUSSION THREAD

**Discussion by osa1** [sourcefrog/cargo-mutants#417] [General]
Is it possible to run `cargo mutants` with a given test command or executable?

I've searched in https://mutants.rs/ and `cargo mutants --help`, but it looks like it only supports `cargo test` and `cargo nextest`.

How hard would it be to support user commands to run the tests?

**Comment by sourcefrog** [comment]
It's not supported at the moment, but it shouldn't be hard, and someone is looking at it. https://github.com/sourcefrog/cargo-mutants/issues/413
